lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -302,7 +302,7 @@
 }
 
 %forum-nav-thread-wrapper-2-content {
-  @include font-size(11);
+  font-size: 11px;
```

Comment:
We should use one of our defined size variables, or use rem/em. Also 11px is tiny. 16px should be our minimum. If we absolutely need smaller, 14px.